### PR TITLE
[INLONG-4280][Manager] Add params for Iceberg sink

### LINKED ISSUE #4280: [Feature][Manager] Add params for Iceberg sink
### Description

Add Iceberg sink info and load node utils 

### Use case

Add Iceberg sink info and load node utils 

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Title Name: [INLONG-4280][Manager]Add params for Iceberg sink

Fixes #4280 

### Motivation

Add params for the Iceberg sink.

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [X] This change is a trivial rework/code cleanup without any test coverage.

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java [hunk 1]
```diff
@@ -225,6 +231,62 @@ public static HbaseLoadNode createLoadNode(HbaseSinkResponse hbaseSinkResponse)
...
+                catalogType = IcebergConstant.CatalogType.forName("hive");
+                break;
+            case HADOOP:
+                catalogType = IcebergConstant.CatalogType.forName("hadoop");
+                break;
+            case HYBRIS:
+                catalogType = IcebergConstant.CatalogType.forName("hybris");
+                break;
+            default:
+                throw new IllegalArgumentException(String.format("Unsupported catalogType=%s for kafka source",
```
Comment (woofyzhao) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java:261]: not kafka

Comment (LvJiancheng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/IcebergSink.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    @ApiModelProperty("Hive reposity location")
+    private String warehouse;
+
+    @ApiModelProperty("Create table or not")
+    private boolean needCreated;
+
+    @ApiModelProperty("Field definitions for hbase")
+    private List<SinkField> sinkFields;
+
```
Comment (gong) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/IcebergSink.java:71]: not hbase

Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/IcebergSink.java:71]: done

**Comment by healchow** [comment]
Please resolve the conflicts, thanks.

**Review by gong — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**